### ISSUE TITLE
Fission IPFS conflicts with system IPFS

### ISSUE THREAD

**Issue by samuelandert** [fission-suite/fission#415]
# Summary
Bug with the fission-cli

## Problem
I am currently trying to install the new fission-cli, but I am forever stuck at installing dependencies and get terminated errors, when I try to register an app.

### Impact
Not able to use any fission-cli interaction properly

## Solution
?

# Detail

**To Reproduce**
1. brew install fission-suite/fission/fission-cli
2. fission setup
3. successful account creation and login
4. stuck at install step:  installing dependencies...

A. When trying to run: fission app register in a separate terminal tab, it returns IPFS termination errors

**Expected behavior**
Successfully register and deploy with the cli an application

**Screenshots**
![Bildschirmfoto_2020-11-11_um_09 43 22](https://user-images.githubusercontent.com/747161/98808686-da97d580-241c-11eb-934d-40070a8e9de3.png)
![Bildschirmfoto_2020-11-11_um_09 43 40](https://user-images.githubusercontent.com/747161/98808687-dbc90280-241c-11eb-8ccb-7f6da51c2637.png)

**Desktop (please complete the following information):**
- macOS 10.15.7
- fission-cli 2.8.0.0
- homebrew 2.5.9

**Comment by benjaminbollen** [comment]
tried to recreate with verbose, but now it completes
- moved `~/.config/fission` folder
- perhaps it retrieved existing keys from the IPFS fission daemon that interferred ?

```
fission setup --verbose
2020-11-11 16:03:43.487540: [debug] Reading global config.yaml
@(<unknown>:0:0)
2020-11-11 16:03:43.487903: [debug] Reading YAML from "/Users/ben/.config/fission/config.yaml"
@(<unknown>:0:0)
2020-11-11 16:03:43.488007: [debug] Path does not exist
@(<unknown>:0:0)
2020-11-11 16:03:43.488053: [debug] Raised exception: Cannot find file
@(<unknown>:0:0)
2020-11-11 16:03:43.488673: [debug] No cached server DID. Fetching from _did.runfission.com
@(<unknown>:0:0)
2020-11-11 16:03:43.491709: [debug] DID retrieved did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:03:43.492273: [debug] Setup (Options {verboseFlag = True, forceOS = Nothing})
@(<unknown>:0:0)
2020-11-11 16:03:43.492370: [debug] Raised exception: User key already exists
@(<unknown>:0:0)
2020-11-11 16:03:43.492497: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:03:43.492557: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:03:43.492692: [debug] Sending web request
@(<unknown>:0:0)
🕐2020-11-11 16:03:44.168222: [debug] Raised exception: FailureResponse (Request {requestPath = (BaseUrl {baseUrlScheme = Https, baseUrlHost = "runfission.com", baseUrlPort = 443, baseUrlPath = ""},"/user/whoami"), requestQueryString = fromList [], requestBody = Nothing, requestAccept = fromList [text/plain;charset=utf-8], requestHeaders = fromList [("Authorization","Bearer eyJ1YXYiOiIxLjAuMCIsImFsZyI6IkVkMjU1MTkiLCJjdHkiOm51bGwsInR5cCI6IkpXVCJ9.eyJwdGMiOiJBUFBFTkQiLCJuYmYiOjE2MDUxMDY5OTMsInJzYyI6eyJmbG9vZnMiOiIvIn0sImV4cCI6MTYwNTEwNzA1MywiaXNzIjoiZGlkOmtleTp6U3RFVVpOMzRKem9oMVVnUmpuMU4zcG1YZWJpQ0I2ZzNBY3dlM2VQUlNxWDFpbVdpeWliUnRENXhtSkdtb05MIiwicHJmIjpudWxsLCJhdWQiOiJkaWQ6a2V5OnpTdEVacHpTTXRUdDlrMnZzemd2Q3dGNGZMUVFTeUExNVc1QVE0ejNBUjZCeDRlRko1Y3JKRmJ1R3hLbWJtYTQifQ.zF3ggPlEWDmiRdsmOSC01_wnntE_EjpkrT0KKAQkSu_m7ugtOGyCH-6W-pcXiKlKOfVV7SHytfbKJNPF1ojvDw")], requestHttpVersion = HTTP/1.1, requestMethod = "GET"}) (Response {responseStatusCode = Status {statusCode = 404, statusMessage = "Not Found"}, responseHeaders = fromList [("Date","Wed, 11 Nov 2020 15:03:44 GMT"),("Transfer-Encoding","chunked"),("Connection","keep-alive"),("Server","Warp/3.3.10")], responseHttpVersion = HTTP/1.1, responseBody = "User not found"})
@(<unknown>:0:0)
Username: testJoeBen
Email: ben+testfission@mosaiclabs.eu
2020-11-11 16:04:49.236467: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:04:49.236632: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:04:49.236865: [debug] Sending web request
@(<unknown>:0:0)
✅ Registration successful! Head over to your email to confirm your account.
Setting default config...2020-11-11 16:04:59.864650: [debug] Initializing config file
@(<unknown>:0:0)
2020-11-11 16:04:59.864752: [debug] Getting peers remotely
@(<unknown>:0:0)
🕛2020-11-11 16:05:00.002469: [debug] No cached server DID. Fetching from _did.runfission.com
@(<unknown>:0:0)
2020-11-11 16:05:00.024049: [debug] DID retrieved did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:05:00.024509: [debug] Writing YAML file to "/Users/ben/.config/fission/config.yaml"
@(<unknown>:0:0)
2020-11-11 16:05:00.024573: [debug] Writing to /Users/ben/.config/fission/config.yaml
@(<unknown>:0:0)
Installing dependencies...2020-11-11 16:05:00.031166: [debug] Setting up IPFS binary for darwin
@(<unknown>:0:0)
2020-11-11 16:05:00.031267: [debug] Making remote IPFS request to HTTP gateway BaseUrl {baseUrlScheme = Https, baseUrlHost = "ipfs.io", baseUrlPort = 443, baseUrlPath = ""}
@(<unknown>:0:0)
2020-11-11 16:05:02.389181: [debug] Remote IPFS success
@(<unknown>:0:0)
2020-11-11 16:05:02.389252: [debug] Writing IPFS binary to /Users/ben/.config/fission/bin/fission-ipfs
@(<unknown>:0:0)
2020-11-11 16:05:02.389287: [debug] Writing to /Users/ben/.config/fission/bin/fission-ipfs
@(<unknown>:0:0)
2020-11-11 16:05:03.840296: [debug] Running local IPFS
@(<unknown>:0:0)
2020-11-11 16:05:03.840475: [debug] Writing to /Users/ben/.config/fission/tmp/04feb9fccaac85b01580cb1f27ebcef5b9f2f90cce517fd92c827b8890484ce4
@(<unknown>:0:0)
2020-11-11 16:05:03.841795: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:05:03.841891: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:05:03.841947: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:05:03.842675: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:05:03.842788: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:03.894656: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:05:04.895821: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:04.931476: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:05:05.936444: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:05.974537: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:05:06.977790: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:07.015251: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:05:08.019665: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:08.058267: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:05:09.062879: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:09.106795: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:05:10.111924: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:05:10.155092: [debug] ExitSuccess
@(<unknown>:0:0)
2020-11-11 16:05:10.155167: [debug] Placed IPFS daemon in MVar
@(<unknown>:0:0)
2020-11-11 16:05:10.155205: [debug] Running: /Users/ben/.config/fission/bin/fission-ipfs bootstrap add /dns4/node.runfission.com/tcp/4001/ipfs/QmVLEz2SxoNiFnuyLpbXsH6SvjPTrHNMU88vCQZyhgBzgw --timeout=3600s
@(<unknown>:0:0)
2020-11-11 16:05:10.190845: [debug] Running local IPFS
@(<unknown>:0:0)
2020-11-11 16:05:10.190965: [debug] Writing to /Users/ben/.config/fission/tmp/04feb9fccaac85b01580cb1f27ebcef5b9f2f90cce517fd92c827b8890484ce4
@(<unknown>:0:0)
2020-11-11 16:05:10.191406: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:05:10.191449: [debug] IPFS Daemon already running
@(<unknown>:0:0)
2020-11-11 16:05:10.191478: [debug] Running: /Users/ben/.config/fission/bin/fission-ipfs bootstrap add /ip4/3.226.224.78/tcp/4001/p2p/QmPeeeZZXxMBAPxxba7a6ggjDr5jLuD3RAFnmPPcvJ9fMS --timeout=3600s
@(<unknown>:0:0)
2020-11-11 16:05:10.226311: [debug] Running local IPFS
@(<unknown>:0:0)
2020-11-11 16:05:10.226432: [debug] Writing to /Users/ben/.config/fission/tmp/04feb9fccaac85b01580cb1f27ebcef5b9f2f90cce517fd92c827b8890484ce4
@(<unknown>:0:0)
2020-11-11 16:05:10.226892: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:05:10.226933: [debug] IPFS Daemon already running
@(<unknown>:0:0)
2020-11-11 16:05:10.226958: [debug] Running: /Users/ben/.config/fission/bin/fission-ipfs bootstrap add /ip4/3.226.224.78/udp/4001/quic/p2p/QmPeeeZZXxMBAPxxba7a6ggjDr5jLuD3RAFnmPPcvJ9fMS --timeout=3600s
@(<unknown>:0:0)
2020-11-11 16:05:10.322911: [debug] Stopping IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:05:10.334671: [debug] Raised exception: User key already exists
@(<unknown>:0:0)
```

**Comment by benjaminbollen** [comment]
I could replicate now. the difference was: I turned my IPFS desktop daemon back on and as a result it outputs in verbose now

` [debug] IPFS daemon startup appears stuck. Retrying.`

```
(base) aboki5:guide ben$ mv ~/.config/fission/ ~/.config/TEST01_fission
(base) aboki5:guide ben$ fission setup --verbose
2020-11-11 16:30:37.138873: [debug] Reading global config.yaml
@(<unknown>:0:0)
2020-11-11 16:30:37.139310: [debug] Reading YAML from "/Users/ben/.config/fission/config.yaml"
@(<unknown>:0:0)
2020-11-11 16:30:37.139424: [debug] Path does not exist
@(<unknown>:0:0)
2020-11-11 16:30:37.139485: [debug] Raised exception: Cannot find file
@(<unknown>:0:0)
2020-11-11 16:30:37.140810: [debug] No cached server DID. Fetching from _did.runfission.com
@(<unknown>:0:0)
2020-11-11 16:30:37.172143: [debug] DID retrieved did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:30:37.173076: [debug] Setup (Options {verboseFlag = True, forceOS = Nothing})
@(<unknown>:0:0)
2020-11-11 16:30:37.173481: [debug] Writing to /Users/ben/.config/fission/key/machine_id.ed25519
@(<unknown>:0:0)
2020-11-11 16:30:37.175342: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:30:37.175555: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:30:37.176200: [debug] Sending web request
@(<unknown>:0:0)
🕑2020-11-11 16:30:37.914734: [debug] Raised exception: FailureResponse (Request {requestPath = (BaseUrl {baseUrlScheme = Https, baseUrlHost = "runfission.com", baseUrlPort = 443, baseUrlPath = ""},"/user/whoami"), requestQueryString = fromList [], requestBody = Nothing, requestAccept = fromList [text/plain;charset=utf-8], requestHeaders = fromList [("Authorization","Bearer eyJ1YXYiOiIxLjAuMCIsImFsZyI6IkVkMjU1MTkiLCJjdHkiOm51bGwsInR5cCI6IkpXVCJ9.eyJwdGMiOiJBUFBFTkQiLCJuYmYiOjE2MDUxMDg2MDcsInJzYyI6eyJmbG9vZnMiOiIvIn0sImV4cCI6MTYwNTEwODY2NywiaXNzIjoiZGlkOmtleTp6U3RFUmNjSkIzUjNueUVkUnl0OUJLTnFCanVmVUF2emU2S1hhb3RQVEdWVzFLVlpnN0QyR3Q2TGZ0bU5ENTZVIiwicHJmIjpudWxsLCJhdWQiOiJkaWQ6a2V5OnpTdEVacHpTTXRUdDlrMnZzemd2Q3dGNGZMUVFTeUExNVc1QVE0ejNBUjZCeDRlRko1Y3JKRmJ1R3hLbWJtYTQifQ.4mnZkYOBIPYsN59nuTfw2epcOS1BJvQVqHMTIAekhsD58ik5KffundoBy6iHg8FK6NwhePPP5hY7RnlmQqMzCA")], requestHttpVersion = HTTP/1.1, requestMethod = "GET"}) (Response {responseStatusCode = Status {statusCode = 404, statusMessage = "Not Found"}, responseHeaders = fromList [("Date","Wed, 11 Nov 2020 15:30:37 GMT"),("Transfer-Encoding","chunked"),("Connection","keep-alive"),("Server","Warp/3.3.10")], responseHttpVersion = HTTP/1.1, responseBody = "User not found"})
@(<unknown>:0:0)
Username: testJoeBen2
Email: ben+test2@mosaiclabs.eu
2020-11-11 16:31:01.257441: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:31:01.257582: [debug] Loaded Server DID: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:31:01.257744: [debug] Sending web request
@(<unknown>:0:0)
✅ Registration successful! Head over to your email to confirm your account.
Setting default config...2020-11-11 16:31:08.077047: [debug] Initializing config file
@(<unknown>:0:0)
2020-11-11 16:31:08.077125: [debug] Getting peers remotely
@(<unknown>:0:0)
🕚2020-11-11 16:31:08.210199: [debug] No cached server DID. Fetching from _did.runfission.com
@(<unknown>:0:0)
2020-11-11 16:31:08.227682: [debug] DID retrieved did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
@(<unknown>:0:0)
2020-11-11 16:31:08.227809: [debug] Writing YAML file to "/Users/ben/.config/fission/config.yaml"
@(<unknown>:0:0)
2020-11-11 16:31:08.227889: [debug] Writing to /Users/ben/.config/fission/config.yaml
@(<unknown>:0:0)
Installing dependencies...2020-11-11 16:31:08.229247: [debug] Setting up IPFS binary for darwin
@(<unknown>:0:0)
2020-11-11 16:31:08.229369: [debug] Making remote IPFS request to HTTP gateway BaseUrl {baseUrlScheme = Https, baseUrlHost = "ipfs.io", baseUrlPort = 443, baseUrlPath = ""}
@(<unknown>:0:0)
2020-11-11 16:31:12.787151: [debug] Remote IPFS success
@(<unknown>:0:0)
2020-11-11 16:31:12.787251: [debug] Writing IPFS binary to /Users/ben/.config/fission/bin/fission-ipfs
@(<unknown>:0:0)
2020-11-11 16:31:12.787315: [debug] Writing to /Users/ben/.config/fission/bin/fission-ipfs
@(<unknown>:0:0)
2020-11-11 16:31:14.223677: [debug] Running local IPFS
@(<unknown>:0:0)
2020-11-11 16:31:14.223829: [debug] Writing to /Users/ben/.config/fission/tmp/04feb9fccaac85b01580cb1f27ebcef5b9f2f90cce517fd92c827b8890484ce4
@(<unknown>:0:0)
2020-11-11 16:31:14.224906: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:31:14.224965: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:31:14.225015: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:31:14.225650: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:31:14.225718: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:14.336396: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:15.340936: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:15.451678: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:16.453709: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:16.548433: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:17.552800: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:17.656235: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:18.658878: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:18.767330: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:19.771347: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:19.876448: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:20.878440: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:20.981467: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:21.984226: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:22.089615: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:23.092087: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:23.191180: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:24.194653: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:24.302419: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:25.305598: [debug] IPFS daemon startup appears stuck. Retrying.
@(<unknown>:0:0)
2020-11-11 16:31:25.320929: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:31:25.321010: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:31:25.321070: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:31:25.321590: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:31:25.321649: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:25.370170: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:26.373890: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:26.495013: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:27.497871: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:27.594155: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:28.599412: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:28.711424: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:29.717210: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:29.816654: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:30.819124: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:30.928274: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:31.930251: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:32.033811: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:33.038307: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:33.140279: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:34.146026: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:34.258503: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:35.259764: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:35.364347: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:36.369598: [debug] IPFS daemon startup appears stuck. Retrying.
@(<unknown>:0:0)
2020-11-11 16:31:36.379719: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:31:36.379791: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:31:36.379838: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:31:36.380253: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:31:36.380308: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:36.491266: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:37.492628: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:37.598440: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:38.601643: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:38.703906: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:39.704601: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:39.808518: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:40.808885: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:40.917316: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:41.921576: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:42.016970: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:43.021395: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:43.132383: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:44.135702: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:44.235093: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:45.239177: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:45.353651: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:46.358747: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:46.462456: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:47.466823: [debug] IPFS daemon startup appears stuck. Retrying.
@(<unknown>:0:0)
2020-11-11 16:31:47.477677: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:31:47.477780: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:31:47.477850: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:31:47.478234: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:31:47.478289: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:47.584749: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:48.589231: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:48.706278: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:49.709364: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:49.808984: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:50.810242: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:50.905458: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:51.909802: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:52.008154: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:53.009020: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:53.108886: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:54.111316: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:54.216007: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:55.216507: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:55.314257: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:56.319121: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:56.417146: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:57.422768: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:57.531789: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:58.533090: [debug] IPFS daemon startup appears stuck. Retrying.
@(<unknown>:0:0)
2020-11-11 16:31:58.543208: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:31:58.543278: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:31:58.543324: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:31:58.543667: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:31:58.543722: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:58.655571: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:31:59.660264: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:31:59.761816: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:00.767424: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:00.889028: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:01.893559: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:01.997842: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:03.000557: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:03.113812: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:04.117241: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:04.204937: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:05.210257: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:05.320508: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:06.322172: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:06.431325: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:07.433020: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:07.527464: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:08.532495: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:08.639398: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:09.642879: [debug] IPFS daemon startup appears stuck. Retrying.
@(<unknown>:0:0)
2020-11-11 16:32:09.655577: [debug] Starting IPFS daemon
@(<unknown>:0:0)
2020-11-11 16:32:09.655653: [debug] Starting new IPFS Daemon
@(<unknown>:0:0)
2020-11-11 16:32:09.655703: [debug] IPFS_PATH set to: Just "/Users/ben/.config/fission/ipfs"
@(<unknown>:0:0)
2020-11-11 16:32:09.656081: [debug] IPFS daemon started
@(<unknown>:0:0)
2020-11-11 16:32:09.656139: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:09.757817: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:10.762045: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:10.867940: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:11.869546: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:11.973509: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:12.979003: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:13.083771: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:14.085616: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:14.184854: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:15.190700: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:15.291125: [debug] ExitFailure 1
@(<unknown>:0:0)
2020-11-11 16:32:16.296320: [debug] Checking if IPFS daemon is running
@(<unknown>:0:0)
2020-11-11 16:32:16.396051: [debug] ExitFailure 1
@(<unknown>:0:0)
```

**Comment by bmann** [comment]
Confirmed and replicated. Thank you @samuelandert and @benjaminbollen 

The issue is that Fission installs an ipfs daemon of our own and runs and controls it automatically. We have configured it to NOT conflict with a local IPFS daemon, but there is an issue with that config, that will be fixed in an upcoming release :)

If you stop/quit IPFS desktop or `brew services stop ipfs` on MacOS fission-cli should work for you as a work around. I'm also updating the guide with a note.

**Comment by expede** [comment]
I believe that I have a fox on my branch. Will be fixed in 2.10

**Comment by expede** [comment]
I typoed above, but it’s so much better so I’m leaving it 🦊

**Comment by expede** [comment]
I believe that this is fixed in the 2.10.x generation 🚀
